server/src/query/table_provider.rs

```diff
@@ -1,140 +0,0 @@
-/*
```

Comment:
Why deleted?

Reply:
This file is unused now

Reply:
Let's create a separate cleanup PR with details on why files are not used. this change is clearly unrelated to caching.

Reply:
>  This change is clearly unrelated to caching.

This file was being used to combine two execution plans ( remote and memory ), but now because of conditional addition of caching, I had to refactor related code in the schema provider file.

Reply:
Please fix commit message with details